.github/workflows/_build_test.yml

```diff
@@ -107,50 +107,12 @@ jobs:
           name: TestResults
           path: test/suites/**/results
 
-  test-upgrade:
```

Comment:
Would the recreation of upgrade tests be worth a ticket already?

Reply:
What's the reasoning behind removing them? My understanding was the work of moving from the `wikibase-shared` volume to a more limited `wikibase-config` was that that would *enable* proper upgrading, because the mw code would no longer be in a volume that persisted.

Reply:
@rti it wouldn't bother me if we did. I had added a note to https://phabricator.wikimedia.org/T345224  as sorta a "heads-up" that a zoomed-out look at our testing paradigm (landscape) might be coming up soon. I think that is enough. 

@RickiJay-WMDE It's complicated, but the short story is the WIkibase image is pretty different now and the upgrade picture reflected in the Upgrade tests suite (as well as the docs), isn't a functional reflection of what upgrades look like going forward in this branch.

Reply:
If they no longer make sense in their current form, should the upgrade functionality be tested some other way?

Reply:
Yes, for sure. Here is the draft ticket created for that during Sprint Planning today: https://phabricator.wikimedia.org/T364847.